Documentation/concepts.rst

```diff
@@ -143,6 +143,36 @@ The Key-Value (KV) Store is used for the following state:
 To simplify things in a larger deployment, the key-value store can be the same
 one used by the container orchestrater (e.g., Kubernetes using etcd).
 
+Assurances
+==========
+
+If cilium loses connectivity with the KV-Store, it guarantees that:
+
+* Normal networking operations will continue;
+
+* If policy enforcement is enabled, the existing endpoints will still have their
+policy enforced but you will lose the ability to add additional containers that
+belong to security identities which are unknown on the node;
+
+* If services are enabled, you will lose the ability to add additional services
+ / loadbalancers;
+
+* When the connectivity is restored with KV-Store, cilium can take up to 5
+minutes to re-sync the out-of-sync state with the KV-Store.
+
+Cilium will be kept running even in out-of-sync with the KV-Store.
+
+If cilium crashes / removed, it is guaranteed that:
```

Comment:
What does "removed" mean in this context?

Reply:
... the following are guaranteed:

Reply:
Fixed